src/shared.ts

```diff
@@ -138,8 +138,12 @@ export const initStripeConnect = (
   stripePromise: Promise<StripeConnectWrapper>,
   initParams: IStripeConnectInitParams
 ): StripeConnectInstanceExtended => {
+  const clientSecretPromise = initParams.fetchClientSecret();
```

Comment:
Can we wrap this in a `try catch` and return `Promise.reject` in the catch? In the case where the implementation of `fetchClientSecret` is something that throws an error, this method will throw, however what we should do is render an error in embedded, like so:

```
try {
  return initParams.fetchClientSecret();
} catch (error) {
  return Promise.reject(error)
}
```